src/ploomber_core/warnings.py

```diff
@@ -6,4 +10,26 @@ class PloomberDeprecationWarning(FutureWarning):
         Changed superclass from UserWarning to FutureWarning
     """
 
-    pass
+    # @
```

Comment:
This comment can be removed

Reply:
fix in [557b5ad](https://github.com/ploomber/core/pull/65/commits/557b5ad96968747255a5006d186873e84d2e37b8)